code/go/0chain.net/core/transaction/http.go

```diff
@@ -143,13 +142,17 @@ func makeSCRestAPICall(scAddress string, relativePath string, params map[string]
 			return errors.Throw(ErrBadRequest, errorMsg)
 		}
 
-		hashString := hex.EncodeToString(hash.Sum(nil))
-		hashCounters[hashString]++
+		resMaxCounterBody = resBody
```

Comment:
This would only return the last response, and no consensus is actually met. This can be a workaround for the fix. But actually it's hard to have consensus as the sharders could be in different LFB when they receive the request, which means they would give different response.

Reply:
added this PR comment as a `note` in the code.